src/posix/platform/system.cpp

```diff
@@ -197,6 +197,10 @@ void platformSetUp(void)
     SuccessOrDie(otSetStateChangedCallback(gInstance, processStateChange, gInstance));
 #endif
 
+#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE || OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE
+    otNat64SetEnabled(gInstance, true);
```

Comment:
Can we consider moving this some other place?

This method `platformSetUp()` seems to be part of posix platform later, and is setting up all the posix platform layer components.